applications/conceptual-model-editor/src/action/actions-react-binding.tsx

```diff
@@ -422,14 +484,102 @@ function createActionsContext(
       openCreateConnectionDialog(source.externalIdentifier, target.externalIdentifier);
     },
 
-    onCreateConnectionToNothing: (source, position) => {
-      console.log("Application.onCreateConnectionToNothing", { source, position });
+    onCreateConnectionToNothing: (source, flowPosition) => {
+      console.log("Application.onCreateConnectionToNothing", { source, flowPosition });
+      diagram.actions().openCanvasToolbar(source, flowPosition, "EDGE-DRAG-CANVAS-MENU-TOOLBAR");
     },
 
     onSelectionDidChange: (nodes, edges) => {
       console.log("Application.onSelectionDidChange", { nodes, edges });
     },
+    onAnchorNode: (diagramNode) => {
+      alert("Anchoring node");
+    },
+    onShowSelectionActions: (source, flowPosition) => {
+      console.log("Application.onShowSelectionActions", { source, flowPosition });
+      diagram.actions().openCanvasToolbar(source, flowPosition, "NODE-SELECTION-ACTIONS-SECONDARY-TOOLBAR");
+    },
+    onLayoutSelection: () => {
+      alert("TODO: currently does nothing (In future - Opens layouting menu - 3 buttons - alignments + layouting)");
+    },
+    onCreateGroup: () => {
+      alert("TODO: currently does nothing (In future - Creating group)");
+    },
+    onShowExpandSelection: () => {
+      alert("TODO: currently does nothing (In future - Showing expansion dialog)");
+    },
+    onShowFilterSelection: () => {
+      alert("TODO: currently does nothing (In future - Showing filter dialog)");
+    },
+    onCanvasOpenCreateClassDialog: (sourceClassNode, positionToPlaceClassOn) => {
+      diagram.actions().closeCanvasToolbar();
+      // TODO: Maybe all of this should be in action - openCreateClassDialogWithModelDerivedFromClassAction
```

Comment:
This is my bad, I should have mentioned it in the PR review myself. 

This was more of a question for you about code stucture rather than for my future self. Since we are introducing TODOs with my postfix, maybe I should start using another postfix marking these types of TODOs, so if I forget to mention them, it is obvious that they are question and will be resolved within PR based on your idea about code structure.

so maybe the postfix should be something like `TODO PRQuestion:` ?

And to explain why did I put this TODO here, basically I was not sure if all of the action methods within the actions-react-binding.tsx should be always implemented like one-liners (or almost one-liners) - that is just call the .*Action method